pulp_ansible/app/galaxy/v1/tasks.py

```diff
@@ -0,0 +1,184 @@
+import os
+import shutil
+import subprocess
+import tempfile
+
+from pulpcore.plugin.models import Artifact
+from pulpcore.plugin.models import ContentArtifact
+from pulpcore.plugin.tasking import add_and_remove
+
+"""
+from pulp_ansible.app.galaxy.v1.utils import (
+    get_role_version,
+    # get_tag_commit_hash,
+    # get_tag_commit_date,
+    # get_path_role_meta
+)
+"""
+
+# from pulp_ansible.app.galaxy.v1.constants import LEGACY_REPOSITORY_NAME
+
+from pulp_ansible.app.models import AnsibleRepository, Role
+
+# from pulp_ansible.app.tasks.roles import GITHUB_URL
+
+# from galaxy_importer.loaders.content import RoleLoader
+# from galaxy_importer.utils import markup as markup_utils
+
+# from git import GitCommandError
+from git import Repo
+
+
+def legacy_role_import(*args, **kwargs):
```

Comment:
In the final version, I prefer having these be named args, kwargs.

Reply:
yeah, i hope to give it fully written out args/kwargs with typedef and docstrings ... just waiting for us to settle on the ins and outs of these functions.